instruqt/delivering-as-an-appliance/01-specifying-the-cluster/assignment.md

```diff
@@ -1,26 +1,223 @@
+manage the appliance. We refer to the Kubernetes cluster that's included as the
+Embedded Cluster since it's "embedded" with your application. The console to
+isntall and manage your application is called the Admin Console. Under the
+hood, the Admin Console uses Helm to install and upgrade your application.
+
+You release your application as a Kubernetes applicance by releasing a Helm
+chart and some additional configuration on the Replicated Platform. The moost
+important file is the configuration for the Embedded Cluster, which can very
+simple. All it requires is the version of the cluster to use.
+
+```
+apiVersion: embeddedcluster.replicated.com/v1beta1
+kind: Config
+spec:
+  version: [[ Instruqt-Var key="EMBEDDED_CLUSTER_VERSION" hostname="shell" ]]
+```
+
+That specification lets the Replicated Vendor Portal know which version of
+Kubernetes to embed. There are [more options for the
+configuration](https://docs.replicated.com/reference/embedded-config), but
+that's all you need to get started.
+
+The Replicated Release
+======================
+
+The Replicated Platform distributes your software to your customers. To do
+this, it needs to know about your application, it's customers, and the files
+you're shipping to them. We talk about those files as a release.
+
+Every release is built around a Helm chart, and that's all it needs. If you've
+completed the [Distributing Your Application with
+Replicated](https://play.instruqt.com/replicated/tracks/distributing-with-replicated)
+lab, you built a release around a Helm chart and installed it using Helm tools.
+To use other installation methods supported by the Reoplicated Platform, like
+the Kubernetes appliance model we're working on, you supplement the Helm chart
+with configuration files like the one we saw above.
+
+We need a minimum of two configuration files: the Embedded Cluster
+configuration and a file that describes which Helm chart to install. We'll put
+those in a directroy with the Helm chart and create a release from those files.
+
+Our Initial Appliance
+=====================
+
+Let's create a simple Kubernetes appliance for Slackernews and release it with
+the Platform. We're going to add our Helm chart and two configuration files to
+the `release` directory. These are the bare minimum set of files we need to
+create the appliance.
+
+### Bumping the Chart Version
+
+Strictly speaking, we don't have to bump the version of our Helm chart since we
+haven't changed it. It's a good practice to increment it, though, since we want
+to have consistent versions between the Helm release and the Replicated
+release. That way, regardless of the install mechanism your uses choose the
+versions will be consistent.
+
+This lab uses the application [Slackernews](https://slackernews.io) and the
+Helm sources for it are in the `slackernews` directory. You can open
+`slackernews/Chart.yaml` in the "Release Editor" tab and update the `version`.
```

Comment:
Am I supposed to do this now? What should I change it to? not clear.

Reply:
Has to be 0.6.0 I think

Reply:
good catch